slack-buggybot/models/bug.rb

```diff
@@ -27,7 +27,7 @@ def self.in_event(event_id)
     end
 
     def self.user_existing_bug(user_id:, event_id:)
-      Bug.in_event(event_id).where(completed: nil).where(assignee: user_id).first
+      Bug.done_in_event(event_id).where(completed: nil).where(assignee: user_id).first
```

Comment:
Not sure how to run / test this locally!

Reply:
No worries, I can test it later 👍